azure/template.json

```diff
@@ -335,6 +345,30 @@
                     }
                 }
             }
+        },
+        {
+            "apiVersion": "2021-04-01",
+            "name": "[concat(variables('apiAppServiceName'), '-', parameters('utcValue'))]",
```

Comment:
do you reckon having a -sub- or -sk- to represent subscription key make the deployment name abit clearer